### DISCUSSION TITLE
Any schedule to support image enhancement tasks

### DISCUSSION THREAD

**Discussion by YinengXiong** [open-mmlab/mmediting#1211] [General]
Any schedule to support image enhancement / photo retouching tasks?

**Comment by zengyh1900** [comment]
Hi @YinengXiong 

Of course, we always want to support more image enhancement/ photo retouching tasks!
However, there're too many features waiting for us to support but we have limited time. 
Would you like to contribute the new models to MMEditing? We're glad to provide any help we can! :star_struck:
